adage-front/src/app/components/OffersSearch/Offers/OfferSummary/OfferSummary.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import format from 'date-fns/format';
+import { ADRESS_TYPE, OfferType } from 'utils/types';
+
+import { ReactComponent as SubcategoryIcon } from "assets/subcategory.svg"
+import { ReactComponent as DateIcon } from "assets/date.svg"
+import { ReactComponent as LocationIcon } from "assets/location.svg"
+import { ReactComponent as EuroIcon } from "assets/euro.svg"
+import { ReactComponent as UserIcon } from "assets/user.svg"
+import { ReactComponent as BuildingIcon } from "assets/building.svg"
+
+import './OfferSummary.scss'
+
+const OfferSummary = ({offer}: {offer: OfferType}): JSX.Element => {
+    const { subcategoryLabel, stocks, venue, extraData } = offer;
+    const { beginningDatetime, numberOfTickets, price} = stocks[0];
+
+    let offerVenue = `${venue.postalCode}, ${venue.city}`;
+
+    if (extraData?.offerVenue) {
+        switch(extraData?.offerVenue.addressType) {
+            case ADRESS_TYPE.OTHER:
+                offerVenue= extraData.offerVenue.otherAddress;
+                break;
+            case ADRESS_TYPE.SCHOOL:
+                offerVenue = "Dans l'établissement scolaire"
```

Comment:
`MAJ`+`option`+`4` sur mac pour faire une apostrophe typographique
```suggestion
                offerVenue = 'Dans l’établissement scolaire'
```